src/optionsdialog.cpp

```diff
@@ -191,15 +229,13 @@ QString OptionsDialog::analysisDescription(int level)
     switch (level)
     {
     case 0:
-        return tr("-");
+        return tr("No analysis");
     case 1:
-        return tr("Minimum");
+        return tr("Auto-Analisis (aaa)");
     case 2:
-        return tr("Basic");
+        return tr("Auto-Analysis Experimental (aaaa)");
```

Comment:
I left the (aaaa) and (aaa) to help people understand the kind of analysis that is being made in r2, so if anyone wants to use the console, they know which command to use.